query-service-impl/build.gradle.kts

```diff
@@ -63,4 +83,9 @@ dependencies {
   testImplementation("org.mockito:mockito-junit-jupiter:3.8.0")
   testImplementation("org.apache.logging.log4j:log4j-slf4j-impl:2.17.1")
   testImplementation("com.squareup.okhttp3:mockwebserver:4.9.3")
+  constraints {
```

Comment:
we'd never use constraints on test dependencies - who is bringing in junit 4?

Reply:
being a test dependency, I can remove it.

Reply:
removed